src/payment/strategies/paypal-commerce/paypal-commerce-payment-processor.ts

```diff
@@ -0,0 +1,62 @@
+import { MissingDataError, MissingDataErrorType } from '../../../common/error/errors';
+
+import { OverlayPaypal } from './index';
+
+const modalWidth = 450;
+const modalHeight = 600;
+
+export default class PaypalCommercePaymentProcessor {
+    private _window = window;
+    private _popup?: Window | null;
+
+    constructor(
+        private _overlay: OverlayPaypal
+    ) {}
+
+    paymentPayPal(approveUrl: string): Promise<any> {
+        return new Promise((resolve, reject) => {
+            const closeWindow = (isResolve: boolean) => {
+                this._overlay.remove();
+
+                if (this._popup) {
+                    this._popup.close();
+                }
+
+                isResolve
+                    ? resolve(true)
```

Comment:
`resolve()`
I don't think you care about the boolean? If it resolves, it means it succeeded.